api/common/modelstatus.go

```diff
@@ -117,5 +117,11 @@ func constructModelStatus(m names.ModelTag, owner names.UserTag, r params.ModelS
 			HAPrimary:   mm.HAPrimary,
 		}
 	}
+	result.Applications = make([]base.Application, len(r.Applications))
+	for k, app := range r.Applications {
+		result.Applications[k] = base.Application{
+			Name: app.Name,
+		}
+	}
```

Comment:
Also, there was a reason we couldn't just use a slice of strings for result.Applications right? Remind me what that was?

Reply:
If I understand the question correctly, we did It because we decided that we (potentially) could need more data in the future than just App.Name, and tried to follow the same approach as with machines.